### ISSUE TITLE
Improve the look of the GUI using CSS

### ISSUE THREAD

**Issue by HEdingfield** [BrightSpots/rcv#140]
Tutorials 25-27 in this playlist give some examples:
https://www.youtube.com/playlist?list=PL6gx4Cwl9DGBzfXLWLSYVy8EbTdpGbUIG

**Comment by chughes297** [comment]
@freedomcounts Would this sort of update count as de minimis? Seems really valuable to do but if we can do it after certification and not worry about it, seems best to do after that cert.

**Comment by freedomcounts** [comment]
This would probably be a de minimis.  If anyone disagrees let me know.

Keith Long, PMP
Freedom Counts, LLC
Keith.long@freedomcounts.com
843-540-1491

**Comment by tarheel** [comment]
Yeah -- if anything at all qualifies as de minimis, it's something like this.
